web-app/CKS.Web/Startup.cs

```diff
@@ -77,7 +93,7 @@ public void ConfigureDevelopment(IApplicationBuilder app, IWebHostEnvironment en
 		{
 			// Serve static files straight for Gatsby's public folder when running locally.
 			// This means you don't have to copy the Gatsby output into the wwwroot folder like we do on TeamCity.
-			var localGatsbyFileProvider = new PhysicalFileProvider(Path.Combine(Directory.GetCurrentDirectory(), "../../gatsby/public"));
+			var localGatsbyFileProvider = new PhysicalFileProvider(Path.Combine(Directory.GetCurrentDirectory(), Configuration.GetValue<string>("GatsbyOutputFolder")));
```

Comment:
We shouldn't need this app setting now that we've set the test environment. Please can you revert?

Reply:
Yes, I realised this, however, its good practice to remove hardcoded into configuration files right?

Reply:
generally but in this case it's *always* one level up as the gatsby folder is always in the same place and npm run build always outputs to the public folder so making it configurable just complicates things